src/openzaak/components/besluiten/models.py

```diff
@@ -195,10 +196,12 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         # save previous zaak for triggers
-        # self._previous_zaak = self.zaak
-
-        self._previous_zaak = self._zaak
-        self._previous_zaak_url = self._zaak_url
+        # for virtual models we can't use FK field because of handler assertions
+        if isinstance(self, ProxyMixin):
```

Comment:
This will require some merge-conflict resolving with the Besluiten-list PR.

Reply:
I'll take care of it